packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -93,6 +94,10 @@ exports.sourceNodes = async (
     pluginConfig,
   })
 
+  if (schema && actions.createTypes) {
```

Comment:
Do we need this check? Maybe we make a major upgrade and drop support for older Gatsby? It's kinda moot anyway, you aren't fixing anything for old versions, are you?

Reply:
Right, I already bumped peer dependency and marked PR as breaking change (so major version bump will be needed), so I think it's safe to drop this check